src/frontend/src/lib/components/ui/Alert.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
+
+	export let alertType: 'info' | 'light-warning' = 'info';
+</script>
+
+<div
+	class="mb-4 flex items-start gap-4 rounded-xl px-4 py-3 text-sm font-medium sm:text-base"
+	class:bg-onahau={alertType === 'info'}
+	class:bg-warning-orange={alertType === 'light-warning'}
+>
+	<div
+		class="min-w-5 py-0 sm:py-0.5"
+		class:text-blue-ribbon={alertType === 'info'}
+		class:text-american-orange={alertType === 'light-warning'}
+	>
+		<IconInfo />
+	</div>
+	<div>
```

Comment:
you need this div?

Reply:
it was already there to be honest, that is why i didn't change it; I agree that it is not useful now, but for other levels will allows us to change the text color of the slot